lib/assets/i18n/log/messages_en.json

```diff
@@ -13,9 +13,9 @@
   "ZWED0030I":"Installed plugin: %s Successful: (%s/%s) Attempted: (%s/%s)",
   "ZWED0031I":"Server is ready at %s, Plugins successfully loaded: %s% (%s/%s)",
   "ZWED0032I":"Server is ready at %s, Plugins successfully loaded: %s% (%s/%s)",
-  "ZWED0033I":"The http port given to the APIML is: %s",
-  "ZWED0034I":"The https port given to the APIML is: %s",
-  "ZWED0035I":"The zlux-apiml config are: %s",
+  "ZWED0033I":"The http port given to the APIML is: ",
```

Comment:
Is excluding the parameter for these errors the way to do things? Did I miss some kind of conversation

If it is, it should probably be removed from the others, I guess. Otherwise this seems unnecessary

Reply:
Read some of my first comments, this is a trick we learned about how best to print objects.